components/patient-data/impl/src/main/resources/ApplicationResources.properties

```diff
@@ -1,232 +1,233 @@
-phenotips.UIXField.reportsHistory=Reports
-phenotips.UIXField.reportsHistory.sheetTitle=Previous medical reports
-phenotips.UIXField.requiredMeasurements=Required measurements warning
-phenotips.UIXField.requiredMeasurements.warning=It is recommended to record weight, height and head circumference for all patients.
-phenotips.UIXField.variants=Copy number variants
-
-phenotips.UIXSection.diagnosisInfo=Diagnosis
-phenotips.UIXSection.familyInfo=Family history and pedigree
-phenotips.UIXSection.filesInfo=Additional files
-phenotips.UIXSection.genotype=Genotype information
-phenotips.UIXSection.measurementInfo=Measurements
-phenotips.UIXSection.medicalHistory=Medical history
-phenotips.UIXSection.patientInfo=Patient information
-phenotips.UIXSection.phenotypeInfo=Clinical symptoms and physical findings
-phenotips.UIXSection.prenatalInfo=Prenatal and perinatal history
-phenotips.UIXSection.variantInfo=Copy number variants
-
-PhenoTips.VariantClass_reference_genome=Reference genome
-PhenoTips.VariantClass_chr=Chr
-PhenoTips.VariantClass_band=Cytoband
-PhenoTips.VariantClass_start=Start
-PhenoTips.VariantClass_end=End
-PhenoTips.VariantClass_type=Type
-PhenoTips.VariantClass_type_del=Deletion
-PhenoTips.VariantClass_type_dup=Duplication
-PhenoTips.VariantClass_classification=Classification
-PhenoTips.VariantClass_classification_none=Unclassified
-PhenoTips.VariantClass_classification_sv=Clinically significant
-PhenoTips.VariantClass_comments=Comments
-
-phenotips.vcf.title=VCF file upload
-phenotips.vcf.sheetTitle=Variants
-PhenoTips.VCF_filename=VCF file
-PhenoTips.VCF_reference_genome=Reference genome
-
-PhenoTips.PatientConsent_real=I confirm that the data entered in this form corresponds to a real patient. (required)
-PhenoTips.PatientConsent_message=The sharing of a broad phenotype description or disease name using standardized terms and/or candidate gene does not require patient consent as deemed by the <a target="_blank" href="http://genomicsandhealth.org/">Global Alliance for Genomics and Health</a>. If you have received patient consent to share additional data, please select the relevant fields below. You may use the <a target="_blank" href="{0}">PhenomeCentral consent form</a> or your own consent form if it allows for the appropriate level of sharing.
-PhenoTips.PatientConsent_genetic=I confirm that consent has been obtained to share this patient's genetic data on restricted access databases.
-PhenoTips.PatientConsent_share_history=I confirm that consent has been obtained to share this patient's medical and family history on restricted access databases.
-PhenoTips.PatientConsent_share_images=I confirm that consent has been obtained to share this patient's medical images/photos on restricted access databases.
-PhenoTips.PatientConsent_matching=This patient is matchable through the MatchmakerExchange and you may receive occasional match notifications (unless you set the visibility to Private).
```

Comment:
Hmm, why is git showing so many changes? Did you use other line endings? This doesn't look like a good idea as it is really hard to check what really has been changed....